### PR TITLE
How to run a community design meeting?

### PR DESCRIPTION
**Pull request by jasonblais**
Adding you Katie as well, in case you have any tips/notes to share based on your individual review meetings with customers for LDAP group sync and guest accounts.

### CODE REVIEW

File: source/process/community-guidelines.rst [hunk 1]
```diff
@@ -29,18 +26,68 @@ On-boarding
...
 Off-boarding
-~~~~~~~~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^^^^^^^
 
 1. When a campaign is finished, offer suggestions for other campaigns or projects they might be interested in.
 
+How to run a community design meeting?
+---------------------------------------------------------
+
+Invite various Mattermost communities to attend a meeting to review the approach
```
Comment (amyblais) [inline source/process/community-guidelines.rst]: This sentence seems incomplete and/or is missing a period.

File: source/process/community-guidelines.rst [hunk 2]
```diff
@@ -29,18 +26,68 @@ On-boarding
...
+^^^^^^^^^^^^^^^^^^^^^^^^
+1. Avoid blindspots in design by reviewing use cases with user communities.
+2. Confirm proposed approach for design aligns with solving the problem discussed.
+3. Brainstorm ideas for solving open design questions.
+
+Preparations
+^^^^^^^^^^^^^^^^^^^^^^^^
+1. Research existing solutions to understand potential use cases brought up by communities.
+2. Draft a proposed design for the problem - this can be a rough outline, more concrete mockups or final design.
+3. Decide two meetings times: one for European and Asian communities, another for North and South American communities.
```
Comment (amyblais) [inline source/process/community-guidelines.rst]: meetings > meeting

File: source/process/community-guidelines.rst [hunk 3]
```diff
@@ -29,18 +26,68 @@ On-boarding
...
+
+ - Avoid regional holidays
+ - Aim for typical working hours in the regions
+ 
+4. Prepare agenda.
+
+ - Introductions of Mattermost team part of the meeting
```
Comment (amyblais) [inline source/process/community-guidelines.rst]: team > team members

File: source/process/community-guidelines.rst [hunk 4]
```diff
@@ -29,18 +26,68 @@ On-boarding
...
+ - Review of use cases
+ - Review of proposed design approach
+ - Open discussion and Q&A
+ - Actions and decisions
+ - Review of next steps
+
+5. Create a public meeting link everyone have permission to access, such as a Zoom link.
```
Comment (amyblais) [inline source/process/community-guidelines.rst]: have > has

**Comment by jasonblais** [comment]
Thanks for the review! Updated

**Pull request merged**